ens-de-lyon-centre-d-ingenierie-documentaire.csl

```diff
@@ -39,297 +35,339 @@
       </term>
     </terms>
   </locale>
+  <!-- /INFORMATIONS -->
+  <!--_-_-_-_-_-SECTION DEFINITION DES MACROS-_-_-_-_-_-_-->
+  <!-- MACRO AUTEUR, EDITEUR SCIENTIFIQUE (ndbp) -->
   <macro name="author">
     <choose>
-      <if variable="author">
-        <names variable="author">
-          <name form="long" and="text" sort-separator=" " initialize-with="." font-style="normal">
+      <if type="interview">
+        <names variable="interviewer">
+          <name form="long" and="text" delimiter-precedes-last="never" sort-separator=" " initialize-with="." font-style="normal">
```

Comment:
I know they were in the style before, but pretty much none of the `font-style="normal"` attributes are necessary. Could you just delete them all?